### DISCUSSION TITLE
How to get standard error for the coefficients in scikit-fda linear regression?

### DISCUSSION THREAD

**Discussion by MRanka29** [GAA-UAM/scikit-fda#611] [Functional Data Analysis discussion]
Hello all,

I am trying to fit function on scalar regression in python using scikit-fda package. The equivalent package in R is fosr. In fosr I get the standard error for the coefficients. Can anyone help me understand how I can get the standard error for the coefficients in python as well? Thanks in Advance

**Comment by vnmabus** [comment]
As far as I remember, we have not implemented that functionality. May I ask what do you want exactly, and why do you need these standard error estimates?
